src/Workleap.OpenApi.MSBuild/OasdiffManager.cs

```diff
@@ -59,9 +59,19 @@ public async Task RunOasdiffAsync(IReadOnlyCollection<string> openApiSpecFiles,
             this._loggerWrapper.LogMessage("- Specification file path: {0}", MessageImportance.High, baseSpecFile);
             this._loggerWrapper.LogMessage("- Specification generated from code path: {0} \n", MessageImportance.High, generatedSpecFilePath);
             
-            var result = await this._processWrapper.RunProcessAsync(oasdiffExecutePath, new[] { "diff", baseSpecFile, generatedSpecFilePath, "--exclude-elements", "description,examples,title,summary", "-f", "text", "-o" }, cancellationToken);
-            this._loggerWrapper.LogMessage(result.StandardOutput, MessageImportance.High);
-            if (!string.IsNullOrEmpty(result.StandardError))
+            var result = await this._processWrapper.RunProcessAsync(oasdiffExecutePath, new[] { "diff", baseSpecFile, generatedSpecFilePath, "--exclude-elements", "description,examples,title,summary", "-o" }, cancellationToken);
+            if (string.IsNullOrEmpty(result.StandardError))
+            {
+                var isChangesDetected = result.ExitCode != 0;
+                this._loggerWrapper.LogMessage("Exit code {0}", MessageImportance.Normal, result.ExitCode);
```

Comment:
I think "oasdiff returned: {0}" would be better than just "exit code {0}" for clarity (or something like that that mentions oasdiff